lib/help_scout.rb

```diff
@@ -18,9 +20,14 @@ def initialize(api_key)
   def create_conversation(data)
     post("conversations", { body: data })
 
-    # Extract ID of created conversation from the Location header
-    conversation_uri = last_response.headers["location"]
-    conversation_uri.match(/(\d+)\.json$/)[1]
+    if last_response.code == 201
```

Comment:
I think it would be nice for readability to extract these status codes into constants, I always forget what they are, or at least I have to think a bit for non 200/403/404 etc.